packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -761,12 +761,15 @@ export class AppStudioPluginImpl {
   > {
     let tabEndpoint, tabDomain;
     if (isArmSupportEnabled()) {
-      if (localDebug) {
```

Comment:
What's the difference between local-debug and remote?

Reply:
local debug is out of provision stage. It will use ctx config other than arm output.
Actually, this method will be called in provision, local debug and validate manifest command.
It uses this condition to separate local debug from other scenarios.